utils/contributors.js

```diff
@@ -59,6 +59,13 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+    {
+        handle_name: 'jp',
+        name: "First Card",
+        image: "/images/jp.webp",
+        description: "Hello everyone, this is the jp card. Go ahead!",
```

Comment:
LGTM but maybe you should try a different "name" and "description" for your card.